signalmlb.lua

```diff
@@ -0,0 +1,40 @@
+-- A simple Lua module for HAProxy that sends signals to the marathon-lb process
+
+function run(cmd)
+  local file = io.popen(cmd)
+  local output = file:read('*a')
+  local success, _, code = file:close()
+  return output, success, code
+end
+
+function send_response(applet, code, response)
+  applet:set_status(code)
+  applet:add_header("content-length", string.len(response))
+  applet:add_header("content-type", "text/plain")
+  applet:start_response()
+  applet:send(response)
+end
+
+core.register_service("signalmlbhup", "http", function(applet)
```

Comment:
I'm not sure how best to do this with HAProxy.. can do something like this (I think? HAProxy manual is long and complicated):
```
  acl signalmlbhup path /_mlb_signal/hup
  http-request deny 405 if !METH_POST signalmlbhup
  http-request use-service lua.signalmlbhup if signalmlbhup
  acl signalmlbusr1 path /_mlb_signal/usr1
  http-request deny 405 if !METH_POST signalmlbusr1
  http-request use-service lua.signalmlbusr1 if signalmlbusr1
```

Like I said in [this comment](https://github.com/mesosphere/marathon-lb/pull/312#issuecomment-248925919), HAProxy is not *ideal* for doing this, and the other endpoints don't limit their valid methods.